lib/swagger_client/version.rb

```diff
@@ -11,5 +11,5 @@
 =end
 
 module SwaggerClient
-  VERSION = '1.0.17'
+  VERSION = '1.0.18'
```

Comment:
oh i think its still 1.0.17 sorry my last update bumped it, you can just leave it as .17 none of that stuff went out yet